test/modules/core/lib/pick-layers.spec.js

```diff
@@ -0,0 +1,450 @@
+          getFillColor: () => [255 * Math.random(), 0, 0],
+          getLineColor: [0, 0, 0, 255],
+          getLineDashArray: [20, 0],
+          getWidth: 20,
+          getElevation: () => Math.random() * 1000,
+          opacity: 0.3,
+          pickable: true,
+          lineDashJustified: true,
+          elevationScale: 0.6
+        })
+      ],
+      views: [new MapView()],
+      viewState: VIEW_STATE,
+      useDevicePixels: false
+    },
+    pickingMethods: {
+      pickObject: [
+        {
+          parameters: {
+            x: 60,
+            y: 160
+          },
+          results: {
+            count: 1
+          }
+        },
+        {
+          parameters: {
+            x: 10,
+            y: 10
+          },
+          results: {
+            count: 0
+          }
+        }
+      ],
+      pickObjects: [
+        {
+          parameters: {
+            x: 300,
+            y: 300,
+            width: 100,
+            height: 100
+          },
+          results: {
+            count: 3
+          }
+        },
+        {
+          parameters: {
+            x: 10,
+            y: 10,
+            width: 10,
+            height: 10
+          },
+          results: {
+            count: 0
+          }
+        }
+      ],
+      pickMultipleObjects: [
+        {
+          parameters: {
+            x: 250,
+            y: 273
+          },
+          results: {
+            count: 1
+          }
+        },
+        {
+          parameters: {
+            x: 10,
+            y: 10
+          },
+          results: {
+            count: 0
+          }
+        }
+      ]
+    }
+  },
+  {
+    id: 'pathLayer',
+    props: {
+      width: 500,
+      height: 550,
+      layers: [
+        new PathLayer({
+          data: DATA.zigzag,
+          opacity: 0.6,
+          getPath: f => f.path,
+          getColor: [128, 0, 0],
+          getWidth: 10,
+          getDashArray: [20, 0],
+          widthMinPixels: 1,
+          pickable: true
+        })
+      ],
+      views: [new MapView()],
+      viewState: VIEW_STATE,
+      useDevicePixels: false
+    },
+    pickingMethods: {
+      pickObject: [
+        {
+          parameters: {
+            x: 260,
+            y: 300
+          },
+          results: {
+            count: 1
+          }
+        },
+        {
+          parameters: {
+            x: 10,
+            y: 10
+          },
+          results: {
+            count: 0
+          }
+        }
+      ],
+      pickObjects: [
+        {
+          parameters: {
+            x: 0,
+            y: 0,
+            width: 400,
+            height: 400
+          },
+          results: {
+            count: 3
+          }
+        },
+        {
+          parameters: {
+            x: 10,
+            y: 10,
+            width: 10,
+            height: 10
+          },
+          results: {
+            count: 0
+          }
+        }
+      ],
+      pickMultipleObjects: [
+        // {
```

Comment:
Why is this commented out?

Reply:
there are two cases involving pathLayer.pickMultipleObjects. Currently it returns the wrong info. We decided to temporarily comment them out and uncomment those cases once the pickMultipleObjects in pathLayer is fixed.